core_gems.rb

```diff
@@ -1,13 +1,16 @@
 dir 'core_gems'
 download "json", "2.1.0"
-download "msgpack", "1.2.2"
-download "cool.io", "1.5.3"
+download "msgpack", "1.2.9"
```

Comment:
Are these version bumps required for building on Windows? If not, can we please pull them out into a separate PR?

Reply:
Yes, they are required.